.circleci/config.yml

```diff
@@ -0,0 +1,52 @@
+SHARED:
+  &installExosphereStep
+  run:
+    name: install exosphere
+    command: |
+      curl --output exo --location --fail https://github.com/Originate/exosphere/releases/download/v0.35.0/exo-linux-amd64
+      chmod +x exo
+      sudo mv exo /usr/local/bin/
+
+version: 2
+jobs:
+  test:
+    working_directory: ~/edx
+    docker:
+      - image: circleci/buildpack-deps
+    steps:
+      - checkout
+      - setup_remote_docker
+      - *installExosphereStep
+      - run:
+          name: run tests
+          command: exo test
+  deploy:
```

Comment:
Do we need to deploy on CI?

Reply:
I think it will be a nice test as it resembles a real project.